scripts/run-integration-tests.sh

```diff
@@ -16,6 +16,7 @@ source "$DIR"/lib/cluster.sh
 # Variables used in /lib/aws.sh
 OS=$(go env GOOS)
 ARCH=$(go env GOARCH)
+GO111MODULE=on
 
 : "${AWS_DEFAULT_REGION:=us-west-2}"
 : "${K8S_VERSION:=1.14.6}"
```

Comment:
This is fine. Might be good to test on a later K8s version, though. EKS currently supports 1.16.8, for instance.